src/textual/screen.py

```diff
@@ -157,7 +157,7 @@ class Screen(Generic[ScreenResultType], Widget):
     title: Reactive[str | None] = Reactive(None, compute=False)
     """Screen title to override [the app title][textual.app.App.title]."""
 
-    COMMAND_SOURCES: ClassVar[set[type[CommandSource]]] = set()
+    COMMAND_SOURCES: ClassVar[set[type[Source]]] = set()
     """The [command sources](/api/command_palette/) for the screen."""
```

Comment:
API link should be of the type `[text here][textual.module.Object]`

Reply:
I think Dave's intent was to link to the page in the API and not a specific object. How else would you do that?

Reply:
Sounds about right to me: if I recall correctly the intent was that it eventually settles on somewhere where the concept of command sources, what they are, how to use them, how to build them, etc; not on a specific bit of class documentation.

Reply:
`/api/xxx` pages can be linked to with `[textual.xxx]`, e.g.:

```suggestion
    """The [command sources][textual.command] for the screen."""
```

If you don't want to switch to that link style, then at least the link requires fixing:

```suggestion
    """The [command sources](/api/command/) for the screen."""
```